srcpkgs/gnome-remote-desktop/template

```diff
@@ -1,18 +1,20 @@
 # Template file for 'gnome-remote-desktop'
 pkgname=gnome-remote-desktop
-version=45.1
+version=46.2
 revision=1
 build_style=meson
+build_helper="gir"
```

Comment:
Why did you add this? I don't see GIR/Typelib generation used in this package?

Reply:
oh i saw something about polkit-gobject stuff
but that was resolved by `polkit-devel`, right, i'll remove it again